src/reducers/fileTree.tsx

```diff
@@ -43,19 +43,19 @@ const recursiveSortInPlace = (node: DirectoryNode): void => {
   });
 };
 
-const getVersionName = (version: Version) => {
-  return getLocalizedString(version.addon.name);
+const getVersionName = (versionAddonName: LocalizedStringMap) => {
+  return getLocalizedString(versionAddonName);
 };
```

Comment:
Why did you change this function? I like that it gives the name of version based on a version object. Now, all it does is returning a localized string for `version.addon.name`, which could be done either directly (if only used in `getRootPath`) or done in a `getAddonName` function instead.

Reply:
But I think I like this `getVersionName` function because it encapsulates the logic for retrieving a good version name.

Reply:
I'm a bit confused. Have you answered all of your own questions above? I changed the function because of the changes to `getRootPath`.

Reply:
Sorry, I like the original version of `getVersionName`. If you can't pass a version to it anymore, I don't think it makes a lot of sense to keep such a function.